css/main.css

```diff
@@ -0,0 +1,1014 @@
+  font-style: normal;
+}
+
+a:link,
+a:hover,
+a:active,
+a:visited {
+  text-decoration: none;
+}
+
+.for-desktop {
+  display: none;
+}
+
+.for-phone {
+  display: block;
+}
+
+button {
+  cursor: pointer;
+}
+
+.transition {
+  transition: all 0.3s ease-in-out 0.1s;
+}
+
+.Event-cover-transition {
+  transition: all 0.1s ease-in-out 0.5s;
+}
+
+.header {
+  background-color: rgba(211, 211, 211, 0.2549019608);
+  background-image: url("../resources/background-image.jpg");
+  background-position: center;
+}
+
+.header .page-header {
+  width: 100%;
+}
+
+.header .page-header .nav-bar {
+  height: inherit;
+  padding: 15px 0 0 20px;
+}
+
+.header .page-header .nav-bar .hamburger {
+  width: 50px;
+}
+
+.header .page-header-body {
+  padding: 0 20px 0 20px;
+  padding-bottom: 10px;
+}
+
+.header .page-header-body .header-text-head {
+  font-weight: 495;
+  line-height: 20px;
+  color: #ec5242;
+  padding-top: 30px;
+}
+
+.header .page-header-body .header-text-body {
+  font-weight: 900;
+  background-image: linear-gradient(360deg, #ec5242, rgba(236, 82, 66, 0.7333333333));
+  color: transparent;
+  margin-top: -15px;
+  font-size: 30px;
+}
+
+.header .page-header-body .header-text-description-container {
+  background-color: #fff;
+  padding: 3px;
+}
+
+.header .page-header-body .header-text-description-container .header-text-description {
+  background-color: rgba(211, 211, 211, 0.2549019608);
+  margin: 0;
+  padding: 10px 30px 10px 20px;
+  color: rgba(39, 42, 49, 0.7803921569);
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 12px;
+}
+
+.header .page-header-body .search-event {
+  background-image: linear-gradient(360deg, #ec5242, rgba(236, 82, 66, 0.7333333333));
+  padding: 10px;
+  color: #fff;
+  width: fit-content;
+  border-radius: 20px;
+  font-weight: 400;
+  margin-left: 30px;
+  display: flex;
+  border: 0;
+  margin-top: 10px;
+}
+
+.header .page-header-body .search-event .enter-arrow {
+  width: 20px;
+  margin-left: 10px;
+}
+
+.main {
+  margin: 0;
+  background-color: #272a31;
+  padding: 20px 5px 30px 5px;
+}
+
+.main .main-head {
+  display: flex;
+  justify-content: center;
+  color: #fff;
+  font-weight: 400;
+}
+
+.main .red-line {
+  color: #ec5242;
+  display: flex;
+  justify-content: center;
+  margin-top: -20px;
+}
+
+.main .top-specials-container {
+  padding: 20px 8px 10px 8px;
+  display: grid;
+  gap: 15px;
+}
+
+.main .top-specials-container .top-specials-item {
+  padding: 10px 8px 15px 8px;
+  background-color: rgba(211, 211, 211, 0.137254902);
+  display: flex;
+}
+
+.comic {
+  padding: 20px 15px 20px 15px;
+  width: 80px;
+  height: auto;
+}
+
+.main .top-specials-container .top-specials-item .number {
+  padding: 0 0 0 8px;
+  color: #ec5242;
+  font-weight: 400;
+  width: 25%;
+  display: flex;
+  align-items: center;
+  padding-left: 20px;
+}
+
+.main .top-specials-container .top-specials-item .special-info-div {
+  /* height: fit-content; */
+  width: 50%;
```

Comment:
- [ ] Kindly delete any commented code as it conflicts with the best practices and having a clean code.

Reply:
Done 👍 @Ofarouq310